crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -213,36 +216,49 @@ impl<'flox> Publish<'flox, NixAnalysis> {
     #[allow(dead_code)] // until consumed by cli
     /// Check whether store paths are substitutable by a given substituter and
     /// return the associated metadata.
+    ///
+    /// If substituter is None, the local store will be used, which is probably
+    /// only useful for testing.
```

Comment:
I agree with the "only useful for testing" part
As mentioned below i also think it can cause invalid entries, i'd therefore like to push this one up a bit